ww4ff/model/inference.py

```diff
@@ -15,115 +15,74 @@
 
 
 class InferenceEngineSettings(BaseSettings):
```

Comment:
Can we keep my version to match the browser version?

Reply:
I'll try out both again, but this wasn't working too well. Maybe my settings weren't optimal.

Reply:
So, I did have noticed that the metrics were too good with the new version.
was that what you mean by working too well?

Reply:
No, it was just observations using the live demo.